feat(ui5-tabcontainer, ui5-list): add events for reordering items by mouse

dimovpetar opened the pull request:
Properties and events will be documented with another PR.

BGSOFUIRODOPI-3189

LidiyaGeorgieva reviewed (commented):
I tested  noticed the following:

**For the list** (http://localhost:8080/packages/main/test/pages/ListDragAndDrop.html) 
1. If I drag "Spain" over the "Germany" and then drag "Spain" on the level of "Bulgaria" (revert nesting) the "Germany" is empty (as expected), but stays grey:
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/9c524e46-baa2-4b4a-9c9c-67fa07de6c9e)

**For the tab-container** (http://localhost:8080/packages/main/test/pages/TabContainerDragAndDrop.html):
2. If I open the right overflow and try to move "Twenty Four" above "Twenty Three" I see the correct cursor (the hand) and the element becomes grey, but when I'm out of the elements boundaries it doesn't move 
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/ba90b34e-9b87-41e1-98e3-cdea486bae6f)
3. (Maybe related with the previous one?) When I drag "Fourteen" over "Thirteen" and nest it it works as expected. But if I try to move "Fourteen" back to its previous place I can't drag it  (as described before - looks like it can be dragged, but it doesn't move):
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/358b2bce-04dd-4add-b3cf-0071a8bda67f)
[update] I noticed that in this case if I start dragging over the padding (not the text) it works, but it is still strange user experience to not work if the element is grabbed in the center (over the text)
4. In tab-container .focus() after drop not always work well
5. There are differences with the design
   5.1. The background of the dragged tab is transparent:
   design:
   ![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/18344a34-1733-4f5e-9a8c-a87a4cb2c63b)
   implementation:
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/05916895-19ae-4ad5-aaf2-e3df6b3bcabb)
   5.2. The circle of the needle is cut
   design:
   ![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/7915eee4-7dce-42ce-b6db-562c904b0e7d)
   implementation:
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/64218f0c-60e2-4881-8444-69807548296d)
   5.3. When I start dragging element the colors and the focus is not as specified
design:
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/2b351dfa-d29a-47e2-bc63-566355c1261e)
implementation:
![image](https://github.com/SAP/ui5-webcomponents/assets/7242508/1e2a2155-7414-4397-93f5-f396ecbcbe53)

dimovpetar reviewed (commented):
TODO: Rename from "DropPlacement" to "Placement"

alexandar-mitsev commented on packages/main/test/pages/TabContainerDragAndDrop.html:
```diff
@@ -0,0 +1,159 @@
+			e.preventDefault();
+		});
+
+		tc.addEventListener("ui5-move", (e) => {
```
the event "move" sounds a bit odd on its own. How about one of those:
move-done (+1 from me)
move-complete (+1 from Teo)
move-finish
move-ready
move-receive

vladitasev replied:
Hello,

I still believe just `move` is the best because it matches the rest of the events: `click`, `change` etc. and none of them are called `click-done` or `change-ready`.

One idea that comes to mind is to have an **event detail** in the future, if necessary, that describes the state of the event outcome, so the app can check: 

`if (e.detail.status === "failed") { // move operation failed` 

or something like this. Probably it would be easier for them to subscribe to just one event (`ui5-move`) and check the outcome in one event handler and do the respective work, rather than subscribing to `ui5-move-complete` and `ui5-move-failed` but I'm not 100% sure.

In the end, I don't have a strong opinion. I'd be fine with any of the options. both `move-done` and `move-complete` sound good to, if you decide to go with a two-word event name.

alexandar-mitsev replied:
Hi,
Ok, the discussion started from that we have drag-end and drop - which is a canceled drop and a successful drop, therefore we were thinking we need to mark the move as successful move.
But you are right - click, change are just single words.

Lets keep it just "move" then.

Thanks

dimovpetar replied:
I will keep it "move" then.

alexandar-mitsev commented on packages/main/src/TabContainer.ts:
```diff
@@ -478,6 +519,148 @@ class TabContainer extends UI5Element {
+	}
+
+	_onPopoverListMove(e: CustomEvent<ListMoveEventDetail>) {
+		const { destination } = e.detail;
```
there is a rare bug (low prio):
- you select a tab
- drag the selected tab in the overflow
- drop it there
- the selected tab is now in the overflow, but since the selected tab must be visible in the header - it automatically pops out of the overflow
- but then the popover still stays open, while it will be better to close

dimovpetar replied:
I couldn't find solution for this problem, because: 1) we need to keep the popover open, in order to allow for rearranging its items and 2) we need to close the popover, if the selected item is put inside it. Both things should happen `onAfterRendering`.
The only solution I could think of is to disable dropping the selected item inside the popover. I have sent a question to the UX designer.

nnaydenow commented on packages/main/src/DropIndicator.ts:
```diff
@@ -0,0 +1,164 @@
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
```
switch to markdown format for JSDOC

dimovpetar replied:
Done

dimovpetar commented:
> I tested noticed the following:
> 
> **For the list** (http://localhost:8080/packages/main/test/pages/ListDragAndDrop.html)
> 
> 1. If I drag "Spain" over the "Germany" and then drag "Spain" on the level of "Bulgaria" (revert nesting) the "Germany" is empty (as expected), but stays grey:
>    ![image](https://private-user-images.githubusercontent.com/7242508/308171852-9c524e46-baa2-4b4a-9c9c-67fa07de6c9e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3MTg1Mi05YzUyNGU0Ni1iYWEyLTRiNGEtOWM5Yy02N2ZhMDdkZTZjOWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjAyYjRiN2NmZmE2MTI3ZGQwN2RjY2MxYTg1ODBkZmYwMWI5MTgxYWE5MDgzMTBjZGFlZWE4ZTU5YjBkODgyZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.3ByDjA3UhFWOOn8ptGFFxZCWveXERMecbWDRpDEHMwk)
> 
> **For the tab-container** (http://localhost:8080/packages/main/test/pages/TabContainerDragAndDrop.html): 2. If I open the right overflow and try to move "Twenty Four" above "Twenty Three" I see the correct cursor (the hand) and the element becomes grey, but when I'm out of the elements boundaries it doesn't move ![image](https://private-user-images.githubusercontent.com/7242508/308173589-ba90b34e-9b87-41e1-98e3-cdea486bae6f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3MzU4OS1iYTkwYjM0ZS05Yjg3LTQxZTEtOThlMy1jZGVhNDg2YmFlNmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjI2MGNjZjFlZTkxZWQ5ODRiYTIzZWI3MWNkODNiOGI1ZTNmYTk3Y2IxOWExYzY1NDc4YjU1ZDk3MzE4MTFiMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.A0YUoxBRfggH0r8-GMJPcWJjU5Pi8b2mdG3IY_QekC8)
> 
> 3. (Maybe related with the previous one?) When I drag "Fourteen" over "Thirteen" and nest it it works as expected. But if I try to move "Fourteen" back to its previous place I can't drag it  (as described before - looks like it can be dragged, but it doesn't move):
>    ![image](https://private-user-images.githubusercontent.com/7242508/308176095-358b2bce-04dd-4add-b3cf-0071a8bda67f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3NjA5NS0zNThiMmJjZS0wNGRkLTRhZGQtYjNjZi0wMDcxYThiZGE2N2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjI0YTJlZjU3NjFkN2E2NmY0Mzk1YWUzOTBiNGNjNWVkMjRiY2MwYmJlMWQ2NzI0ZDlhZDhjOTk0YWQyZTUxMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6Ku2TRpxsyvNXoBjTtlru_IdMkynqrckAFJc1-qJHfM)
>    [update] I noticed that in this case if I start dragging over the padding (not the text) it works, but it is still strange user experience to not work if the element is grabbed in the center (over the text)
> 4. In tab-container .focus() after drop not always work well
> 5. There are differences with the design
> 
> 5.1. The background of the dragged tab is transparent: design: ![image](https://private-user-images.githubusercontent.com/7242508/308178568-18344a34-1733-4f5e-9a8c-a87a4cb2c63b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3ODU2OC0xODM0NGEzNC0xNzMzLTRmNWUtOWE4Yy1hODdhNGNiMmM2M2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmQxNTA0YTRiZDhlMmYwMTZjNjk1YTFmOWI2Njg0YTI3ZWU1YTg3YTExZDhmN2E1ZmQ0NzQ2ODAzOWU4YTk3NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.x56g9blpqCY31EBF5rt0AzXCa5v15B_HcegHS2D4a2M) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308178626-05916895-19ae-4ad5-aaf2-e3df6b3bcabb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3ODYyNi0wNTkxNjg5NS0xOWFlLTRhZDUtYWFmMi1lM2RmNmIzYmNhYmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzdhM2E5OTQ2NWJlYzU3ZGU4NWQ1M2E1OWJlZjFlZmExMmI1ZjdiMDEzOWEwNjllNTU0ZDE1ZmI0ZjZjYzVkYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.QqbLlIm2n5ZxMo4HI4EOsF_A6IAPsfJz_ehSVsXR6RM)
> 
> 5.2. The circle of the needle is cut design: ![image](https://private-user-images.githubusercontent.com/7242508/308179452-7915eee4-7dce-42ce-b6db-562c904b0e7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3OTQ1Mi03OTE1ZWVlNC03ZGNlLTQyY2UtYjZkYi01NjJjOTA0YjBlN2QucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTAzZDNiNDRkM2QyZTJmZWQ3ZTUxY2QxNjg4Y2MyZjQyYWVmYTkzMDFmYTRmNTkwMWQ4MTEyNzFiMTk5MWNjZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oXgAACkyqqXEU15JbJ28K0pOyWr8U6qUcIoO2IcJTkY) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308179499-64218f0c-60e2-4881-8444-69807548296d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3OTQ5OS02NDIxOGYwYy02MGUyLTQ4ODEtODQ0NC02OTgwNzU0ODI5NmQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODBmNDY3YTY2ZDE3MmVhYjA3ZTE4MGUxMWZhYjI3M2JmMzhhYzA3ZTQ2NTRmZWQ2YmY5ZWViMzM2ZWIwNzM0OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.QD4_oIJbxoGNrxTSg8NbKQQo0Ptmemd1fyIZCTxEkXg)
> 
> 5.3. When I start dragging element the colors and the focus is not as specified design: ![image](https://private-user-images.githubusercontent.com/7242508/308181918-2b351dfa-d29a-47e2-bc63-566355c1261e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE4MTkxOC0yYjM1MWRmYS1kMjlhLTQ3ZTItYmM2My01NjYzNTVjMTI2MWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjM1NjQ5YTk4NDliYmUyOTEzNjEyYmYzNTg3NTU3NDUyMzE5YTBlODY2OTQxNTdjYzYzMGMzZmVlNDRkZWQ2YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.CTZSJUiNY2vN0kUVYanXU7z_L0Trwl010UatJRF3t9w) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308181952-1e2a2155-7414-4397-93f5-f396ecbcbe53.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE4MTk1Mi0xZTJhMjE1NS03NDE0LTQzOTctOTNmNS1mMzk2ZWNiY2JlNTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjNiZGM5NTVhZThmNjFiZGUxNDE1MDQ4ZTVkYTYyMzg3OTA3MjFiZjc4MDVjZjM2ODZmYzcxNGNmY2YwYTNhNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.EgkR5FQkxmj5dDI3jxBs9fWEhhlC4D3limjMXBSM-Mw)

1. Fixed 
2. Issue that you can' drag the list items by their text, which in this case is slotted element - seems like bug or feature in Chromium based browsers. See https://issues.chromium.org/issues/328263459. I will wait to see if they will fix it, as there is no workaround I can  think of at the moment
3. Same as 2.
4. Will be discussed (TODO)
5. 
   1. fixed
   2. fixed
   3. fixed

dimovpetar commented:
> TODO: Rename DropPlacement" to "Placement"

Done

dimovpetar commented:
> If in "ui5-move-over" the "e.preventDefault();" is not called, the needle shouldn't be shown.

Fixed

LidiyaGeorgieva commented:
> > I tested noticed the following:
> > **For the list** (http://localhost:8080/packages/main/test/pages/ListDragAndDrop.html)
> > 
> > 1. If I drag "Spain" over the "Germany" and then drag "Spain" on the level of "Bulgaria" (revert nesting) the "Germany" is empty (as expected), but stays grey:
> >    ![image](https://private-user-images.githubusercontent.com/7242508/308171852-9c524e46-baa2-4b4a-9c9c-67fa07de6c9e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3MTg1Mi05YzUyNGU0Ni1iYWEyLTRiNGEtOWM5Yy02N2ZhMDdkZTZjOWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjAyYjRiN2NmZmE2MTI3ZGQwN2RjY2MxYTg1ODBkZmYwMWI5MTgxYWE5MDgzMTBjZGFlZWE4ZTU5YjBkODgyZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.3ByDjA3UhFWOOn8ptGFFxZCWveXERMecbWDRpDEHMwk)
> > 
> > **For the tab-container** (http://localhost:8080/packages/main/test/pages/TabContainerDragAndDrop.html): 2. If I open the right overflow and try to move "Twenty Four" above "Twenty Three" I see the correct cursor (the hand) and the element becomes grey, but when I'm out of the elements boundaries it doesn't move ![image](https://private-user-images.githubusercontent.com/7242508/308173589-ba90b34e-9b87-41e1-98e3-cdea486bae6f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3MzU4OS1iYTkwYjM0ZS05Yjg3LTQxZTEtOThlMy1jZGVhNDg2YmFlNmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjI2MGNjZjFlZTkxZWQ5ODRiYTIzZWI3MWNkODNiOGI1ZTNmYTk3Y2IxOWExYzY1NDc4YjU1ZDk3MzE4MTFiMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.A0YUoxBRfggH0r8-GMJPcWJjU5Pi8b2mdG3IY_QekC8)
> > 
> > 3. (Maybe related with the previous one?) When I drag "Fourteen" over "Thirteen" and nest it it works as expected. But if I try to move "Fourteen" back to its previous place I can't drag it  (as described before - looks like it can be dragged, but it doesn't move):
> >    ![image](https://private-user-images.githubusercontent.com/7242508/308176095-358b2bce-04dd-4add-b3cf-0071a8bda67f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3NjA5NS0zNThiMmJjZS0wNGRkLTRhZGQtYjNjZi0wMDcxYThiZGE2N2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjI0YTJlZjU3NjFkN2E2NmY0Mzk1YWUzOTBiNGNjNWVkMjRiY2MwYmJlMWQ2NzI0ZDlhZDhjOTk0YWQyZTUxMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6Ku2TRpxsyvNXoBjTtlru_IdMkynqrckAFJc1-qJHfM)
> >    [update] I noticed that in this case if I start dragging over the padding (not the text) it works, but it is still strange user experience to not work if the element is grabbed in the center (over the text)
> > 4. In tab-container .focus() after drop not always work well
> > 5. There are differences with the design
> > 
> > 5.1. The background of the dragged tab is transparent: design: ![image](https://private-user-images.githubusercontent.com/7242508/308178568-18344a34-1733-4f5e-9a8c-a87a4cb2c63b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3ODU2OC0xODM0NGEzNC0xNzMzLTRmNWUtOWE4Yy1hODdhNGNiMmM2M2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmQxNTA0YTRiZDhlMmYwMTZjNjk1YTFmOWI2Njg0YTI3ZWU1YTg3YTExZDhmN2E1ZmQ0NzQ2ODAzOWU4YTk3NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.x56g9blpqCY31EBF5rt0AzXCa5v15B_HcegHS2D4a2M) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308178626-05916895-19ae-4ad5-aaf2-e3df6b3bcabb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3ODYyNi0wNTkxNjg5NS0xOWFlLTRhZDUtYWFmMi1lM2RmNmIzYmNhYmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzdhM2E5OTQ2NWJlYzU3ZGU4NWQ1M2E1OWJlZjFlZmExMmI1ZjdiMDEzOWEwNjllNTU0ZDE1ZmI0ZjZjYzVkYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.QqbLlIm2n5ZxMo4HI4EOsF_A6IAPsfJz_ehSVsXR6RM)
> > 5.2. The circle of the needle is cut design: ![image](https://private-user-images.githubusercontent.com/7242508/308179452-7915eee4-7dce-42ce-b6db-562c904b0e7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3OTQ1Mi03OTE1ZWVlNC03ZGNlLTQyY2UtYjZkYi01NjJjOTA0YjBlN2QucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTAzZDNiNDRkM2QyZTJmZWQ3ZTUxY2QxNjg4Y2MyZjQyYWVmYTkzMDFmYTRmNTkwMWQ4MTEyNzFiMTk5MWNjZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oXgAACkyqqXEU15JbJ28K0pOyWr8U6qUcIoO2IcJTkY) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308179499-64218f0c-60e2-4881-8444-69807548296d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3OTQ5OS02NDIxOGYwYy02MGUyLTQ4ODEtODQ0NC02OTgwNzU0ODI5NmQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODBmNDY3YTY2ZDE3MmVhYjA3ZTE4MGUxMWZhYjI3M2JmMzhhYzA3ZTQ2NTRmZWQ2YmY5ZWViMzM2ZWIwNzM0OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.QD4_oIJbxoGNrxTSg8NbKQQo0Ptmemd1fyIZCTxEkXg)
> > 5.3. When I start dragging element the colors and the focus is not as specified design: ![image](https://private-user-images.githubusercontent.com/7242508/308181918-2b351dfa-d29a-47e2-bc63-566355c1261e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE4MTkxOC0yYjM1MWRmYS1kMjlhLTQ3ZTItYmM2My01NjYzNTVjMTI2MWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjM1NjQ5YTk4NDliYmUyOTEzNjEyYmYzNTg3NTU3NDUyMzE5YTBlODY2OTQxNTdjYzYzMGMzZmVlNDRkZWQ2YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.CTZSJUiNY2vN0kUVYanXU7z_L0Trwl010UatJRF3t9w) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308181952-1e2a2155-7414-4397-93f5-f396ecbcbe53.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE4MTk1Mi0xZTJhMjE1NS03NDE0LTQzOTctOTNmNS1mMzk2ZWNiY2JlNTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjNiZGM5NTVhZThmNjFiZGUxNDE1MDQ4ZTVkYTYyMzg3OTA3MjFiZjc4MDVjZjM2ODZmYzcxNGNmY2YwYTNhNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.EgkR5FQkxmj5dDI3jxBs9fWEhhlC4D3limjMXBSM-Mw)
> 
> 1. Fixed
> 2. Issue that you can' drag the list items by their text, which in this case is slotted element - seems like bug or feature in Chromium based browsers. See https://issues.chromium.org/issues/328263459. I will wait to see if they will fix it, as there is no workaround I can  think of at the moment
> 3. Same as 2.
> 4. Will be discussed (TODO)
> 5. 1. fixed
>    2. fixed
>    3. fixed

1. Text color is OK now, but the background of "Germany" stays grey (as it is hovered):
<img width="287" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/7242508/6abc8c4d-2359-4a82-8242-12021bdef8fe">
5.  There are light grey vertical borders visible (for Horizon Light/Dark and Fiori 3 Light/Dark themes):
<img width="150" alt="image" src="https://github.com/SAP/ui5-webcomponents/assets/7242508/622cb06d-15cc-443b-a510-fdfe640b7547">

dimovpetar commented:
> > > I tested noticed the following:
> > > **For the list** (http://localhost:8080/packages/main/test/pages/ListDragAndDrop.html)
> > > 
> > > 1. If I drag "Spain" over the "Germany" and then drag "Spain" on the level of "Bulgaria" (revert nesting) the "Germany" is empty (as expected), but stays grey:
> > >    ![image](https://private-user-images.githubusercontent.com/7242508/308171852-9c524e46-baa2-4b4a-9c9c-67fa07de6c9e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3MTg1Mi05YzUyNGU0Ni1iYWEyLTRiNGEtOWM5Yy02N2ZhMDdkZTZjOWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjAyYjRiN2NmZmE2MTI3ZGQwN2RjY2MxYTg1ODBkZmYwMWI5MTgxYWE5MDgzMTBjZGFlZWE4ZTU5YjBkODgyZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.3ByDjA3UhFWOOn8ptGFFxZCWveXERMecbWDRpDEHMwk)
> > > 
> > > **For the tab-container** (http://localhost:8080/packages/main/test/pages/TabContainerDragAndDrop.html): 2. If I open the right overflow and try to move "Twenty Four" above "Twenty Three" I see the correct cursor (the hand) and the element becomes grey, but when I'm out of the elements boundaries it doesn't move ![image](https://private-user-images.githubusercontent.com/7242508/308173589-ba90b34e-9b87-41e1-98e3-cdea486bae6f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3MzU4OS1iYTkwYjM0ZS05Yjg3LTQxZTEtOThlMy1jZGVhNDg2YmFlNmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjI2MGNjZjFlZTkxZWQ5ODRiYTIzZWI3MWNkODNiOGI1ZTNmYTk3Y2IxOWExYzY1NDc4YjU1ZDk3MzE4MTFiMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.A0YUoxBRfggH0r8-GMJPcWJjU5Pi8b2mdG3IY_QekC8)
> > > 
> > > 3. (Maybe related with the previous one?) When I drag "Fourteen" over "Thirteen" and nest it it works as expected. But if I try to move "Fourteen" back to its previous place I can't drag it  (as described before - looks like it can be dragged, but it doesn't move):
> > >    ![image](https://private-user-images.githubusercontent.com/7242508/308176095-358b2bce-04dd-4add-b3cf-0071a8bda67f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3NjA5NS0zNThiMmJjZS0wNGRkLTRhZGQtYjNjZi0wMDcxYThiZGE2N2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YjI0YTJlZjU3NjFkN2E2NmY0Mzk1YWUzOTBiNGNjNWVkMjRiY2MwYmJlMWQ2NzI0ZDlhZDhjOTk0YWQyZTUxMiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6Ku2TRpxsyvNXoBjTtlru_IdMkynqrckAFJc1-qJHfM)
> > >    [update] I noticed that in this case if I start dragging over the padding (not the text) it works, but it is still strange user experience to not work if the element is grabbed in the center (over the text)
> > > 4. In tab-container .focus() after drop not always work well
> > > 5. There are differences with the design
> > > 
> > > 5.1. The background of the dragged tab is transparent: design: ![image](https://private-user-images.githubusercontent.com/7242508/308178568-18344a34-1733-4f5e-9a8c-a87a4cb2c63b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3ODU2OC0xODM0NGEzNC0xNzMzLTRmNWUtOWE4Yy1hODdhNGNiMmM2M2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmQxNTA0YTRiZDhlMmYwMTZjNjk1YTFmOWI2Njg0YTI3ZWU1YTg3YTExZDhmN2E1ZmQ0NzQ2ODAzOWU4YTk3NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.x56g9blpqCY31EBF5rt0AzXCa5v15B_HcegHS2D4a2M) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308178626-05916895-19ae-4ad5-aaf2-e3df6b3bcabb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3ODYyNi0wNTkxNjg5NS0xOWFlLTRhZDUtYWFmMi1lM2RmNmIzYmNhYmIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzdhM2E5OTQ2NWJlYzU3ZGU4NWQ1M2E1OWJlZjFlZmExMmI1ZjdiMDEzOWEwNjllNTU0ZDE1ZmI0ZjZjYzVkYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.QqbLlIm2n5ZxMo4HI4EOsF_A6IAPsfJz_ehSVsXR6RM)
> > > 5.2. The circle of the needle is cut design: ![image](https://private-user-images.githubusercontent.com/7242508/308179452-7915eee4-7dce-42ce-b6db-562c904b0e7d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3OTQ1Mi03OTE1ZWVlNC03ZGNlLTQyY2UtYjZkYi01NjJjOTA0YjBlN2QucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTAzZDNiNDRkM2QyZTJmZWQ3ZTUxY2QxNjg4Y2MyZjQyYWVmYTkzMDFmYTRmNTkwMWQ4MTEyNzFiMTk5MWNjZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oXgAACkyqqXEU15JbJ28K0pOyWr8U6qUcIoO2IcJTkY) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308179499-64218f0c-60e2-4881-8444-69807548296d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE3OTQ5OS02NDIxOGYwYy02MGUyLTQ4ODEtODQ0NC02OTgwNzU0ODI5NmQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODBmNDY3YTY2ZDE3MmVhYjA3ZTE4MGUxMWZhYjI3M2JmMzhhYzA3ZTQ2NTRmZWQ2YmY5ZWViMzM2ZWIwNzM0OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.QD4_oIJbxoGNrxTSg8NbKQQo0Ptmemd1fyIZCTxEkXg)
> > > 5.3. When I start dragging element the colors and the focus is not as specified design: ![image](https://private-user-images.githubusercontent.com/7242508/308181918-2b351dfa-d29a-47e2-bc63-566355c1261e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE4MTkxOC0yYjM1MWRmYS1kMjlhLTQ3ZTItYmM2My01NjYzNTVjMTI2MWUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjM1NjQ5YTk4NDliYmUyOTEzNjEyYmYzNTg3NTU3NDUyMzE5YTBlODY2OTQxNTdjYzYzMGMzZmVlNDRkZWQ2YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.CTZSJUiNY2vN0kUVYanXU7z_L0Trwl010UatJRF3t9w) implementation: ![image](https://private-user-images.githubusercontent.com/7242508/308181952-1e2a2155-7414-4397-93f5-f396ecbcbe53.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4ODk1MTUsIm5iZiI6MTcwOTg4OTIxNSwicGF0aCI6Ii83MjQyNTA4LzMwODE4MTk1Mi0xZTJhMjE1NS03NDE0LTQzOTctOTNmNS1mMzk2ZWNiY2JlNTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMDhUMDkxMzM1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjNiZGM5NTVhZThmNjFiZGUxNDE1MDQ4ZTVkYTYyMzg3OTA3MjFiZjc4MDVjZjM2ODZmYzcxNGNmY2YwYTNhNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.EgkR5FQkxmj5dDI3jxBs9fWEhhlC4D3limjMXBSM-Mw)
> > 
> > 
> > 
> > 1. Fixed
> > 2. Issue that you can' drag the list items by their text, which in this case is slotted element - seems like bug or feature in Chromium based browsers. See https://issues.chromium.org/issues/328263459. I will wait to see if they will fix it, as there is no workaround I can  think of at the moment
> > 3. Same as 2.
> > 4. Will be discussed (TODO)
> > 5. 1. fixed
> >    2. fixed
> >    3. fixed
> 
> 1. Text color is OK now, but the background of "Germany" stays grey (as it is hovered):
> 
> <img alt="image" width="287" src="https://private-user-images.githubusercontent.com/7242508/311645998-6abc8c4d-2359-4a82-8242-12021bdef8fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyMzU2NDksIm5iZiI6MTcxMDIzNTM0OSwicGF0aCI6Ii83MjQyNTA4LzMxMTY0NTk5OC02YWJjOGM0ZC0yMzU5LTRhODItODI0Mi0xMjAyMWJkZWY4ZmUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTJUMDkyMjI5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTZiYzNiOGJhN2FkYzhkMWVhMDhkY2JmZGU3OTcyNGMzNjk0MjRmYjA2ZjFjMjdjZGM2YjJjZTE5NTk2ODI4OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.__8LRbSFMNCGBQcE22DNwPcgTleuspRRl61i03BCS18">
> 5. There are light grey vertical borders visible (for Horizon Light/Dark and Fiori 3 Light/Dark themes):
> 
> <img alt="image" width="150" src="https://private-user-images.githubusercontent.com/7242508/311646239-622cb06d-15cc-443b-a510-fdfe640b7547.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyMzU2NDksIm5iZiI6MTcxMDIzNTM0OSwicGF0aCI6Ii83MjQyNTA4LzMxMTY0NjIzOS02MjJjYjA2ZC0xNWNjLTQ0M2ItYTUxMC1mZGZlNjQwYjc1NDcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMTJUMDkyMjI5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjllYmNiNzJiZGFkMzQ0ZThhMGMzMzE3ZmQ1NjEyMTc4MjE2YWNhNzJlM2Q5OWJlMGEwMzEyMDcwODQyZDU0NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GwhOUXwOFE-BKetZONlrfvq7tBOuB7dcVaZa7QxzLV4">

1. Looks like :hover pseud class remains active on the wrong element. Opened Chromium issue https://issues.chromium.org/issues/329096331. Anyways, this is not a real case, it it used only for testing in our internal test page
5. Fixed

LidiyaGeorgieva reviewed (commented):
+1 
looks good to me

dimovpetar commented on packages/main/test/pages/ListDragAndDrop.html:
```diff
@@ -0,0 +1,107 @@
+</head>
+
+<body class="list1auto">
+	<ui5-checkbox id="reorder" text="Draggable items" checked></ui5-checkbox>
```
Nope. Fixed it
